### PR TITLE
Recipe for migration JakartaEE Security is wrong

### LINKED ISSUE #160: Recipe for migration JakartaEE Security is wrong
Packages in our codebase were changed from `javax.security.*` to `jakarta.security.*` even though any dependency of `javax.security.enterprise` is included in the project.

The security packages `javax.security.*` are included in the JDK itself, and for JDK 11, it still contains the prefix `javax.*`, so probably these packages should not be updated in the codebase.

If we specify the required packages more, we'll avoid unnecessary collisions.

Issue to OpenRewrite docs: https://github.com/openrewrite/rewrite-docs/issues/111

See: https://github.com/quarkusio/quarkus/issues/30198
https://github.com/keycloak/keycloak/issues/16247

### PR DESCRIPTION
**Pull request by mabartos**
Fixes #160

For more info, see the attached issue.

### CODE REVIEW

File: src/main/resources/META-INF/rewrite/jakarta-ee-9.yml [hunk 1]
```diff
@@ -572,21 +572,21 @@ recipeList:
...
-displayName: Migrate deprecated `javax.security` packages to `jakarta.security`
+displayName: Migrate deprecated `javax.security.enterprise` packages to `jakarta.security.enterprise`
 description: Java EE has been rebranded to Jakarta EE, necessitating a package relocation.
 recipeList:
   - org.openrewrite.maven.AddDependency:
       groupId: jakarta.security.enterprise
       artifactId: jakarta.security.enterprise-api
       version: 2.x
-      onlyIfUsing: javax.security.*
+      onlyIfUsing: javax.security.enterprise.+
```
Comment (mabartos) [inline src/main/resources/META-INF/rewrite/jakarta-ee-9.yml:582]: I'm not sure if use here '+' or '*'.

Comment (jkschneider) [reply src/main/resources/META-INF/rewrite/jakarta-ee-9.yml:582]: It's a '*', will change that and merge.

**Comment by mabartos** [comment]
@jkschneider Thanks!

**Comment by jkschneider** [comment]
Thank you @mabartos. Also thanks for pointing out the `.+` usage as there were several others written that way too.

**Pull request merged**